static/js/message_edit.js

```diff
@@ -904,56 +906,57 @@ export function edit_last_sent_message() {
     });
 }
 
-function hide_delete_btn_show_spinner(deleting) {
-    if (deleting) {
-        $("do_delete_message_button").prop("disabled", true);
-        $("#delete_message_modal > div.modal-footer > button").hide();
-        const delete_spinner = $("#do_delete_message_spinner");
-        loading.make_indicator(delete_spinner, {abs_positioned: true});
-    } else {
-        loading.destroy_indicator($("#do_delete_message_spinner"));
-        $("#do_delete_message_button").prop("disabled", false);
-        $("#delete_message_modal > div.modal-footer > button").show();
-    }
-}
-
 export function delete_message(msg_id) {
     $("#delete-message-error").html("");
-    overlays.open_modal("#delete_message_modal");
+
+    const html_body = render_delete_message_modal_body();
+    const modal_parent = $("#message_feed_container");
```

Comment:
How did you pick this parent element?

Reply:
Closest common ancestor to `all_messages` and `private_messages` view with an id.
Looks like it should have been `$("#main_div")`